ash/src/device.rs

```diff
@@ -1510,7 +1510,7 @@ pub trait DeviceV1_0 {
         size: vk::DeviceSize,
         flags: vk::MemoryMapFlags,
     ) -> VkResult<*mut c_void> {
-        let mut data: *mut c_void = mem::uninitialized();
+        let mut data: *mut c_void = mem::zeroed();
```

Comment:
We should probably use `std::ptr::null_mut` here.

Reply:
Done!